Examples/MAX78000/CNN/actionrecognition-demo/project.mk

```diff
@@ -0,0 +1,36 @@
+# This file can be used to set build configuration
+# variables.  These variables are defined in a file called 
+# "Makefile" that is located next to this one.
+
+# For instructions on how to use this system, see
+# https://github.com/Analog-Devices-MSDK/VSCode-Maxim/tree/develop#build-configuration
+
+# **********************************************************
+
+# Add your config here!
+
+$(info Note: This project is designed and tested for the OV7692 only.)
+override CAMERA=OV7692
+
+# Set a higher optimization level.  The increased performance
+# is required for the CameraIF DMA code to work within the
+# timing requirements of the Parallel Camera Interface.
+MXC_OPTIMIZE_CFLAGS = -O2
+
+# Add some additional directories to the build based on what board
+# we're compiling for...
+ifeq "$(BOARD)" "EvKit_V1"
```

Comment:
There are some TFT-related build errors for the EVKIT.  Do we want to support it for this example, or just support the MAX78000FTHR?